cmd/rqlite/main.go

```diff
@@ -394,10 +395,13 @@ func getHTTPClient(argv *argT) (*http.Client, error) {
 		}
 	}
 
-	client := http.Client{Transport: &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: argv.Insecure, RootCAs: rootCAs},
-		Proxy:           http.ProxyFromEnvironment,
-	}}
+	client := http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: argv.Insecure, RootCAs: rootCAs},
+			Proxy:           http.ProxyFromEnvironment,
+		},
+		Timeout: 10 * time.Second,
```

Comment:
Let's make this a const, like what we have at line 26. E.g. 

`const httpTimeout = 10 * time.Second`

Reply:
Can do.